src/test/java/gov/healthit/chpl/aqa/stepDefinitions/CMSIDsLookupSteps.java

```diff
@@ -0,0 +1,98 @@
+package gov.healthit.chpl.aqa.stepDefinitions;
+import static org.testng.Assert.assertTrue;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import cucumber.api.java.en.Given;
+import cucumber.api.java.en.Then;
+import cucumber.api.java.en.When;
+import gov.healthit.chpl.aqa.pageObjects.CMSidReverseLookupPage;
+import gov.healthit.chpl.aqa.pageObjects.SearchPage;
+
+/**
+ * Class CMSIDsLookupSteps definition.
+ */
+public class CMSIDsLookupSteps {
+
+    private WebDriver driver;
+    private static final int DELAY = 40;
+        
+    public CMSIDsLookupSteps() {
+    driver = Hooks.getDriver();
+    }
+    
+    /**
+     * To load CMS ID Reverse Lookup page 
+     */
+    
+    @Given("^I am an end-user on CMS ID Reverse Lookup page$")
+    public void i_am_an_end_user_on_CMS_ID_Reverse_Lookup_page() {
+        driver.get("https://chpl.ahrqstg.org/#/resources/cms_lookup");
+        driver.manage().window().maximize();
+    }
+
+    /**
+     * Lookup one of the 3 unverifiable CMS IDs on reverse lookup page 
+     */
+    
+   @When("^I look up CMS ID \"(.*)\" in the reverse look-up tool$")
+    public void I_look_up_CMS_ID_in_the_reverse_lookup_tool(final String cmsId){
+        CMSidReverseLookupPage.inputCertificationId(driver).sendKeys(cmsId);
+        CMSidReverseLookupPage.SearchLookupResults(driver).click();
+    }
+    
+    /**
+     * Asserts that listings returned in result are as expected  
+     */
+
+    @Then("^I should see the listings \"(.*)\" that make up the CMS ID$")
+    public void i_should_see_the_listings_that_make_up_the_CMS_ID(final String chplId) throws Throwable {
+        assertTrue(CMSidReverseLookupPage.CertIdResultsTable_CHPLIdcolumn(driver).getText().contains(chplId));
+    }
+    
+    /**
+     * Loads CHPL home/search page  
+     */
+    
+    @Given("^I am an end user on CHPL home page$")
+    public void i_am_an_end_user() throws Throwable {
+        driver.get("https://chpl.ahrqstg.org/#/search");
+        WebDriverWait wait = new WebDriverWait(driver, DELAY);
+        driver.manage().window().maximize();
```

Comment:
I think there's something about the maximize call that breaks stuff. If it's not necessary for the step please remove it